src/detectors/FEMC/FEMC.cc

```diff
@@ -22,30 +22,36 @@ extern "C" {
         using namespace eicrecon;
 
         InitJANAPlugin(app);
-
+        // Make sure digi and reco use the same value
+        decltype(CalorimeterHitDigiConfig::capADC)        EcalEndcapP_capADC = 16384; //16384, assuming 14 bits. For approximate HGCROC resolution use 65536
+        decltype(CalorimeterHitDigiConfig::dyRangeADC)    EcalEndcapP_dyRangeADC = 3 * dd4hep::GeV;
+        decltype(CalorimeterHitDigiConfig::pedMeanADC)    EcalEndcapP_pedMeanADC = 200;
+        decltype(CalorimeterHitDigiConfig::pedSigmaADC)   EcalEndcapP_pedSigmaADC = 2.4576;
+        decltype(CalorimeterHitDigiConfig::resolutionTDC) EcalEndcapP_resolutionTDC = 10 * dd4hep::picosecond;
         app->Add(new JChainMultifactoryGeneratorT<CalorimeterHitDigi_factoryT>(
           "EcalEndcapPRawHits", {"EcalEndcapPHits"}, {"EcalEndcapPRawHits"},
           {
             .eRes = {0.00340 * sqrt(dd4hep::GeV), 0.0009, 0.0 * dd4hep::GeV}, // (0.340% / sqrt(E)) \oplus 0.09%
             .tRes = 0.0,
-            .capADC = 65536, //2^16  (approximate HGCROC resolution) old 16384
-            .capTime = 100, // given in ns, 4 samples in HGCROC
-            .dyRangeADC = 3 * dd4hep::GeV,
-            .pedMeanADC = 100,
-            .pedSigmaADC = 0.7,
-            .resolutionTDC = 10 * dd4hep::picosecond,
+            .threshold = 15 * dd4hep::MeV,
```

Comment:
```suggestion
```

Reply:
This would go to thresholdValue (in adc units)

Reply:
Yes-ish, in the future, the cut on energy will move into the reco class and be entirely optional. But the logic at this moment in time means _this_ is the threshold that is specified in energy in the threshold. I expect most if not all calorimetry groups to set it to 0 and only rely on the adc value, but using the .threshold here is true at face value to the parameters provided.